rust/basic_bitcoin/src/basic_bitcoin/basic_bitcoin.did

```diff
@@ -0,0 +1,38 @@
+type satoshi = nat64;
+
+type bitcoin_address = text;
+
+type block_hash = blob;
+
+type outpoint = record {
+  txid : blob;
+  vout : nat32
+};
+
+type utxo = record {
+  outpoint: outpoint;
+  value: satoshi;
+  height: nat32;
+};
+
+type get_utxos_response = record {
+  utxos: vec utxo;
+  tip_block_hash: block_hash;
+  tip_height: nat32;
+  next_page: opt blob;
+};
+
+service : {
+    "get_p2pkh_address": () -> (bitcoin_address);
+
+    "get_balance": (address: text) -> (satoshi);
+
+    "get_utxos": (bitcoin_address) -> (get_utxos_response);
+
+    "get_current_fee_percentiles": () -> (vec nat64);
+
+    "send": (record {
+      destination_address: text;
+      amount_in_satoshi: nat64;
```

Comment:
Why not `amount: satoshi`?

Reply:
It's subtle. The reason I wrote `amount_in_satoshi` is for clarity in the candid UI. If it's `amount: satoshi` the box will just say "amount (nat64)" and it wouldn't be clear what the unit is.

<img width="770" alt="image" src="https://user-images.githubusercontent.com/208628/177282763-bbca899e-eb75-49c7-9121-70da8962634e.png">